docs/Training-Self-Play.md

```diff
@@ -37,31 +57,68 @@ The ELO calculation (discussed below) depends on this final reward being either
 
 The reward signal should still be used as described in the documentation for the other trainers and [reward signals.](Reward-Signals.md) However, we encourage users to be a bit more conservative when shaping reward functions due to the instability and non-stationarity of learning in adversarial games. Specifically, we encourage users to begin with the simplest possible reward function (+1 winning, -1 losing) and to allow for more iterations of training to compensate for the sparsity of reward.
 
+### Team Change
+
+The `team-change` ***command line argument*** corresponds to the number of *trainer_steps* between switching the learning team. So,
+if you run with the command line flag `--team-change=200000`, the learning team will change every `200000` trainer steps. This ensures each team trains
+for precisely the same number of steps. Note, this is not specified in the trainer configuration yaml file, but as a command line argument.
+
+A larger value of `team-change` will allow the agent to train longer against it's opponents.  The longer an agent trains against the same set of opponents
+the more able it will be to defeat them. However, training against them for too long may result in overfitting to the particular opponent strategies
+and so the agent may fail against the next batch of opponents.
+
+The value of `team-change` will determine how many snapshots of the agent's policy are saved to be used as opponents for the other team.  So, we
+recommend setting this value as a function of the `save_steps` parameter which is discussed in the next section.
+
+Recommended Range : 4x-10x where x=`save_steps`
```

Comment:
Instead of specifying the value here, would it be easier to specify it as a multiple of `save_steps`?

Reply:
I don't have strong feelings either way.  If you think that's more intuitive then that works for me.

Reply:
Not strong feelings, I just think it makes it easier to twist one knob at a time, instead of having to twist 2 in unison.